docs/opa.md

```diff
@@ -1,5 +1,117 @@
 # Access Control Policy with OPA
 
+## Default Policy in Chronicle
+
+The default policy in Chronicle defines the access control rules for
+authorization decisions. It consists of the `allow_transactions` and
+`common_rules` packages, which define the authorization rules for
+transaction.
+
+### Default Policy Description
+
+The default policy in Chronicle is defined using the Rego policy language of
```

Comment:
maybe link to https://www.openpolicyagent.org/docs/latest/policy-language/ or somesuch